scaleway/resource_vpc_public_gateway_dhcp.go

```diff
@@ -53,8 +53,8 @@ func resourceScalewayVPCPublicGatewayDHCP() *schema.Resource {
 			},
 			"enable_dynamic": {
 				Type:        schema.TypeBool,
-				Computed:    true,
 				Optional:    true,
+				Default:     true,
```

Comment:
This has an impact on the pat-rules test. You should just rerun the test to update the cassettes